packages/utils/src/lib/__snapshots__/report-to-stdout.integration.test.ts.snap

```diff
@@ -6,71 +6,71 @@ exports[`report-to-stdout > should contain all sections when using the fixture r
+  constructors
+● Disallow loops with a body that allows only one iteration           0
+● Disallow missing displayName in a React component definition        0
+● Disallow missing React when using JSX                               0
+● Disallow negating the left operand of relational operators          0
 ● Disallow passing of children as props                               0
-● Disallow when a DOM element is using both children and              0
-  dangerouslySetInnerHTML
+● Disallow React to be incorrectly marked as unused                   0
+● Disallow reassigning \`const\` variables                              0
+● Disallow the use of \`debugger\`                                      0
+● Disallow the use of undeclared variables unless mentioned in        0
+  \`/*global */\` comments
+● Disallow undeclared variables in JSX                                0
+● Disallow unescaped HTML entities from appearing in markup           0
 ● Disallow usage of deprecated methods                                0
-● Disallow direct mutation of this.state                              0
 ● Disallow usage of findDOMNode                                       0
 ● Disallow usage of isMounted                                         0
 ● Disallow usage of the return value of ReactDOM.render               0
-● Disallow using string references                                    0
-● Disallow unescaped HTML entities from appearing in markup           0
 ● Disallow usage of unknown DOM property                              0
+● Disallow use of optional chaining in contexts where the             0
+  \`undefined\` value is not allowed
+● Disallow using Object.assign with an object literal as the first    0
+  argument and prefer the use of object spread instead
+● Disallow using string references                                    0
+● Disallow variables used in JSX to be incorrectly marked as unused   0
+● Disallow when a DOM element is using both children and              0
+  dangerouslySetInnerHTML
+● Enforce a maximum number of lines per file                          0
+● Enforce camelcase naming convention                                 0
+● Enforce comparing \`typeof\` expressions against valid strings        0
+● Enforce consistent brace style for all control statements           0
 ● Enforce ES5 or ES6 class for returning value in render function     0
+● enforces the Rules of Hooks                                         0
+● Require \`let\` or \`const\` instead of \`var\`                           0
+● Require calls to \`isNaN()\` when checking for \`NaN\`                  0
+● Require or disallow \\"Yoda\\" conditions                               0
+● Require using arrow functions for callbacks                         0
 
 
 Lighthouse audits
 
 ● First Contentful Paint                                              1.2 s
 ● Largest Contentful Paint                                            1.5 s
```

Comment:
Why isn't Largest Contentful Paint first? Its value is the highest and I cannot see any score.

Reply:
As we found out, the sorting is proper here, but the score is not present in the view.